src/watts/plugin.py

```diff
@@ -125,3 +152,5 @@ def prerun(self, params: Parameters, filename: Optional[str] = None):
         """
         # Render the template
         self.render_template(params, filename=filename)
+        for render_template in self.extra_render_templates:
```

Comment:
We need to add an example with `extra_template_inputs` in the `example/` (based on SAS), and mention this option in the Documentation.

Reply:
This will be done in another MR by @zhieejhia93